package.json

```diff
@@ -62,7 +62,7 @@
 		"eslint-plugin-promise": "6.1.1",
 		"git-branch-is": "4.0.0",
 		"husky": "8.0.3",
-		"jison-gho": "^0.6.1-216",
```

Comment:
Good to see you dived in. There was a security issue with a subdependency - that is why we transitioned to -gho. Looks like its fixed now.